src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -306,4 +306,6 @@ public net.minecraft.world.gen.layer.LayerUtil func_202829_a(JLnet/minecraft/wor
 private-f net.minecraft.world.raid.Raid$WaveMember field_221284_f # VALUES
 public net.minecraft.world.server.ServerChunkProvider field_186029_c # chunkGenerator
 public net.minecraft.world.server.ServerChunkProvider field_73251_h # worldObj
-public net.minecraft.world.storage.FolderName <init>(Ljava/lang/String;)V # constructor
\ No newline at end of file
+public net.minecraft.world.storage.FolderName <init>(Ljava/lang/String;)V # constructor
+public net.minecraft.world.biome.BiomeGenerationSettings <init>(Ljava/util/function/Supplier;Ljava/util/Map;Ljava/util/List;Ljava/util/List;)V
+public net.minecraft.world.biome.MobSpawnInfo <init>(FLjava/util/Map;Ljava/util/Map;Z)V
```

Comment:
Make sure to run the `checkATs` task as I am pretty sure that it will modify this given if memory serves the file should be alphabetical and the `b` in `biome` comes before the `s` in `storage`.